front/lib/api/llm/clients/noop/index.ts

```diff
@@ -32,15 +65,20 @@ export class NoopLLM extends LLM {
     conversation: _conversation,
     prompt: _prompt,
   }: StreamParameters): AsyncGenerator<LLMEvent> {
+    // Emit a simple flow of event:
```

Comment:
adding something a bit more complex for soupinou, because it can be useful to have more than 1 events locally and for tests